readme.txt

```diff
@@ -2,7 +2,7 @@
 Contributors: hyyan, decarvalhoaa, jonathanmoorebcsorg
 Tags: cms, commerce, e-commerce, e-shop, ecommerce, multilingual, products, shop, woocommerce, polylang, bilingual, international, language, localization, multilanguage, multilingual, translate, translation
 Requires at least: 3.8
-Tested up to: 4.7
+Tested up to: 4.7.5
```

Comment:
Patch versions of wordpress won't break the plugin , so let's just use the last wordpress minor version 4.7